client/src/components/sideBar/SideBar.js

```diff
@@ -134,13 +142,13 @@ export function SideBar() {
         className={`fixed left-0 top-0 z-50 flex h-full w-1/5 flex-row overflow-hidden transition-all
                        ${isSidebarOpen ? 'left-0' : 'left-[-20vw]'}`}
       >
-        <div className="flex h-full w-full flex-col justify-between overflow-hidden bg-slate-300 bg-[url('../public/little-prince.jpg')] bg-cover bg-center p-2 bg-blend-soft-light">
```

Comment:
bring back the little prince pls

Reply:
I like the gradient idea, but maybe we can experiment with a different color?